db/seeds.rb

```diff
@@ -1,5 +1,10 @@
 ranks = %i[
-  email_rank location_rank organization_name_rank phone_number_rank website_rank
+  email_rank
+  location_rank
+  organization_name_rank
+  organization_type_rank
```

Comment:
Not so obvious from the reformatted diff, but `organization_type_rank` was the one that was missing.